tor.nim

```diff
@@ -237,18 +239,117 @@ proc connect*(s: ProxySocket, address: string, port: Port) =
   p[0] = cast[char](port.uint16 shr 8)
   p[1] = cast[char](port)
 
+  echo "Connecting through tor"
   s.inner.send("\x05\x01\x00\x03" & address.len.char & address & p)
 
+proc send*(s: ProxySocket; data: pointer; size: int): int = 
+  #[
+    Sends data to a socket.
+    Note: This is a low-level version of send. You likely should use the version below.
+  ]#
+  return s.inner.send(data, size)
+
 proc send*(s: ProxySocket; data: string; flags = {SafeDisconn}) =
-  ## Sends data to a socket
+  #[
+    Sends data to a socket
+  ]#
   s.inner.send(data, flags)
 
+proc trySend*(s: ProxySocket; data: string): bool = 
+  #[
+    Safe alternative to send. Does not raise an OSError when an error occurs, and instead returns false on failure.
```

Comment:
Please remove "Safe"

Reply:
sorry I literaly copied and pasted from the new module